src/fairness/weighted_tree/test/Makefile.am

```diff
@@ -6,8 +6,8 @@ AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS)
 
 AM_CPPFLAGS = -I$(top_srcdir) $(FLUX_CORE_CFLAGS)
 
-TESTS = weighted_tree_test01.t
-check_PROGRAMS = $(TESTS) load_accounting_db
+TESTS = weighted_tree_test01.t load_accounting_db_test02.t
```

Comment:
I believe this commit can be split into at least three:

- input/output data
- test source code
- Makefile changes